spec/reporters/console_reporter_spec.js

```diff
@@ -408,6 +408,29 @@ describe("ConsoleReporter", function() {
     expect(this.out.getOutput()).toContain("Incomplete: not all bars were frobnicated");
   });
 
+  it("reports a summary when done that shows info for a failed spec with no expectations", function() {
```

Comment:
This looks good but it would also be good to have tests that verify that the message is not shown when the spec has failed expectations and when it has passed expectations.